CODEOWNERS

```diff
@@ -169,6 +169,9 @@ Kconfig*                                  @tejlmand
 /subsys/mpsl/                             @rugeGerritsen
 /subsys/net/                              @rlubos
 /subsys/net/lib/azure_*                   @jtguggedal @simensrostad @coderbyheart
+/subsys/net/lib/ftp_client/               @junqingzou
+/subsys/net/lib/zzhc/                     @junqingzou
+/subsys/net/lib/icalendar_parser/         @lats1980
 /subsys/net/lib/lwm2m_client_utils/       @rlubos @VeijoPesonen
```

Comment:
nit: would be good to keep alphabetic order in there

Reply:
Updated, Robert